index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Front-End School</title>
+  <link rel="stylesheet" href="./src/styles.css">
+</head>
+
+<body>
+  <!-- A DIV which contains header & headline background image -->
+  <div class="headline-bg">
+    <header>
+      <div>
+        <ul class="top-header-list">
+          <li><img src="./Images/facebook.png" alt="facebook icon"></li>
+          <li><img src="./Images/twitter.png" alt="twitter icon"></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li class="last"><a href="#">Log out</a></li>
+        </ul>
+      </div>
+
+      <nav>
+        <!-- Desktop Navbar -->
+        <div class="desktop-nav">
+          <img class="page-logo" src="./Images/logo.png" alt="page logo">
+          <ul class="desktop-nav-list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#services">Our Services</a></li>
+            <li><a href="#team-container">Our Team</a></li>
+            <li><a href="#supporters">Sponsors</a></li>
+            <li class="last"><a class="list-btn" href="#">Our Projects</a></li>
+          </ul>
+        </div>
```

Comment:
Good job 👌it looks fantastic, but here is how to improve.
1. It's required to follow the original design colors and sizing, so in your top header list section the background color looks a little bit lighter, please make sure to use the original color.
2. The spacing between the desktop nav list is not the same as it is in the original design, I'd highly recommend increasing it.
3. You're replacing the `about` link with `home` on the about page, instead, I'd recommend adding a link to your logo so the user can visit the home page by clicking on it.
4. You did a great job by adding orange border to the `Our Projects` button on the Navbar, and please make sure to use the same color for button text as well.

Yours: 
![image](https://user-images.githubusercontent.com/34779928/205141073-5b42721b-4ee6-4fea-a840-57ae65de5810.png)

Original:
![image](https://user-images.githubusercontent.com/34779928/205141089-59bf9ac0-1181-4c5b-9645-86ffc1263492.png)